pkg/config/config.go

```diff
@@ -874,6 +874,10 @@ func Default() (*Config, error) {
 	if config != nil || configErr != nil {
```

Comment:
Can we move this down to `defaultConfig()`? The logic belongs together, and I am afraid we might overlook it in future changes.

Reply:
If we leave these here, then we don't need to reset them.

Reply:
Ah, now I get it. Thanks for clarifying!